Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.css

```diff
@@ -24,7 +24,7 @@
  */
 
 .sidebar > .panel.details.style-font > .content .details-section > .content > .group > .row.simple > .label {
-    width: 105px; /* Width of `Historical Figures` or `Optical Size (opsz)`. */
+    min-width: 105px; /* Width of `Historical Figures` or `Optical Size (opsz)`. */
```

Comment:
While we are editing this line, let's fix past me's mistake and end the comment with "in English."